modules/sql-engine/src/integrationTest/sql/types/time/test_time.test

```diff
@@ -1,32 +1,35 @@
 # name: test/sql/types/time/test_time.test
 # description: Test basic TIME functionality
 # group: [time]
-# Ignore https://issues.apache.org/jira/browse/IGNITE-15619
 
 statement ok
 PRAGMA enable_verification
 
 statement ok
-CREATE TABLE times(i TIME)
+CREATE TABLE times(i TIME(3))
 
 statement ok
-INSERT INTO times VALUES ('00:01:20'), ('20:08:10.998'), ('20:08:10.33'), ('20:08:10.001'), (NULL)
+INSERT INTO times VALUES ('00:01:20'), ('20:08:10.998'), ('20:08:10.33'), ('20:08:10.001')
+
+# TODO https://issues.apache.org/jira/browse/IGNITE-15123 Multi-tuple insert fails on validation
+statement ok
+INSERT INTO times VALUES (NULL)
 
 query T
-SELECT * FROM times
+SELECT * FROM times order by i
```

Comment:
instead of adding `ORDER BY` clause, it's better to use `rowsort` option to do comparison. Please refer to javadoc of `ItSqlLogicTest` for details

Reply:
Fixed, thanks.